src/app/project/project.component.html

```diff
@@ -4,16 +4,16 @@
     <div class="breadcrumb-bar columns">
       <a routerLink="/">Home</a>
       <i class="icon ion-ios-arrow-forward"></i>
-      <a routerLink="project/:id">Project ID {{ project.id }}</a>
+      <a routerLink="project/:id">Project ID {{ project?.id }}</a>
```

Comment:
For some reason the unit tests don't seem to prefetch data like the real app, so I had to add these.